src/autocomplete/src/Autocomplete.js

```diff
@@ -176,9 +176,9 @@ export default class Autocomplete extends PureComponent {
     width,
     inputValue,
     highlightedIndex,
-    selectItemAtIndex,
```

Comment:
is this just used internally? not super familiar with the downshift api (and can't super easily find v3 docs)

Reply:
This is exposed externally (as are most downshift props), but it gets captured by `...restDownshiftProps` when we receive props on `<Autocoomplete />` and pass those `...restDownshiftProps` to children as a function.

All that to say that this PR doesn't change the component API we expose, but we don't need to pluck it out explicitly anymore.